test/e2e/common/ttlafterfinished.go

```diff
@@ -36,10 +35,8 @@ const dummyFinalizer = "k8s.io/dummy-finalizer"
 var _ = framework.KubeDescribe("TTLAfterFinished", func() {
 	f := framework.NewDefaultFramework("ttlafterfinished")
 
-	It("Job should be deleted once it finishes after TTL seconds [Feature:TTLAfterFinished]", func() {
-		if !utilfeature.DefaultFeatureGate.Enabled(features.TTLAfterFinished) {
-			framework.Skipf("Skip because %s feature is not enabled; run test with --feature-gates=%s=true", features.TTLAfterFinished, features.TTLAfterFinished)
-		}
+	alphaFeatureStr := "[Feature:TTLAfterFinished]"
+	It(fmt.Sprintf("Job should be deleted once it finishes after TTL seconds", alphaFeatureStr), func() {
```

Comment:
The format string needs a `%s` at the end. Also just curious, what's the reason for breaking it out into a separate var?

Reply:
Fixed. Breaking it out just to make it more obvious that this is tested in the alpha feature suite.